### DISCUSSION TITLE
Is there a better way to detect commonjs wildcard export and better warning prompts?

### DISCUSSION THREAD

**Discussion by skywalker512** [web-infra-dev/rspack#2110] [Q&A] [answered]
In my personal projects, there are many monorepos with many wildcard exports. 

```js
// a.js
export const a = 1
// index.js
export * from './a';
```

They are compiled into CommonJS using TypeScript and packaged with RSPack. However, there are many warnings here, but they are actually usable.

simple example 

## Reproduce
https://github.com/skywalker512/rspack-export-wildcard

pnpm build

<img width="1455" alt="image" src="https://user-images.githubusercontent.com/12960906/224212077-8dc44c20-0582-46a5-8e76-dcaef7ba2cdd.png">

In my personal monorepo project, there are hundreds of warnings, including third-party commonjs libraries, which is very annoying.

**Comment by IWANABETHATGUY** [comment, marked as answer]
Fixed by https://github.com/web-infra-dev/rspack/issues/2111
